### ISSUE TITLE
refactor next steps

### ISSUE THREAD

**Issue by ilackarms** [solo-io/service-mesh-hub#841]
**Required:**
- [ ] docs
  - [x] protobuf documentation (review for typos)
  - [x] changelog summarizing new changes (Istio 1.7, stability, workloads controlled by daemonsets, replicasets, etc.)
  - [ ] Guides
    - [x] Document VM access policy enforcement ingress exception AuthorizationPolicy
    - [ ] Document `.global` suffix requirement for Istio cross-cluster DNS names
    - [x] Update documentation for "Understanding the Shared Root Process", [see here](https://docs.solo.io/service-mesh-hub/latest/guides/federate_identity/#understanding-the-shared-root-process)
- [x] cli
  - [x] check
    - [x] check observed generation & acceptance status of all crds
  - [x] Install
  - [x] Uninstall
  - [x] Cluster
    - [x] Register
    - [x] Deregister
  - [x] Version
  - [x] Demo
    - [x] Istio-multicluster
- [x] tests
  - [x] e2e
    - [x] Istio 1.5
    - [x] Istio 1.6
  - [x] unit tests
- [x] federation / shared trust / csr
  - [x] allow disabling mTLS
- [x] one vmesh per mesh
- [x] deterministic sort Istio VirtualService HTTP Routes
- [x] renames 
  - [x] MeshWorkload to Workload
  - [x] MeshService to TrafficTarget
  - [x] Approver to Applier
  - [x] "master" cluster to "management" cluster
- [x] federated subsets
- [x] merge `upgrades2` into skv2 master, cut release, bump service-mesh-hub to depend on skv2 release

** Nice to have: **
- [ ] CLI
  - [ ] check multicluster connectivity

**Comment by ilackarms** [comment]
meshctl features:
- check registered cluster connectivity
- check observed generation & acceptance status of all crds
- describe to show entire service mesh graph

**Comment by ilackarms** [comment]
dns names suffixed with .global, report dns names to meshservice / failover service status

**Comment by harveyxia** [comment]
`meshctl mesh restart` to restart all pods in the given mesh

**Comment by ilackarms** [comment]
once-over api
